client/components/preview-licence.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export default function NextSteps() {
+  const { isActionable, previewLink } = useSelector(state => state.application);
+  if (!isActionable) {
```

Comment:
Is there some aspect of how this is set that limits it to ASRU users only?

Reply:
It's the same logic as the next steps button, so it shows when you are looking at the version in the task (not asru specific)

Reply:
do we want it to be asru only?

Reply:
The ticket is titled "Give ASRU a preview of the draft licence for PPL applications" so it's somewhat ambiguous. My assumption was that it was meant to be exclusive though.

Reply:
ok cool I'll make it asru only for now - easy to reverse